Week2/homework/maartjes-work.js

```diff
@@ -43,19 +43,28 @@ const tuesday = [
 ];
 
 const maartjesTasks = monday.concat(tuesday);
+
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const mappedTasks = tasks.map(task => task.duration / 60);
+  const filteredTasks = mappedTasks.filter(task => task >= 2);
+  const multiplyTasks = filteredTasks.map(task => task * hourlyRate);
```

Comment:
In this step we are mapping hours to a an 'earning', by multiplying the filtered hours by the hourly rate. Again, the variable and parameter names must tell this story:

```js
const earnings = hours.map(hour => hour * hourlyRate);
```